### PR TITLE
Fallout of PR #1010

### PR DESCRIPTION
**Pull request by bstoeger**
### Describe the pull request:

- [x] Bug fix
- [ ] Functional change
- [ ] New feature
- [x] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

These are two minor cleanups, whereby one is also a bug-fix.

1) Remove the only manipulation of `existing_file` from C-code for a better frontend/backend separation. This fixes a bug introduced in #1010: If in cloud storage and "close file" was chosen, the "cloud online" menu entry stayed active.
2) Remove redundant assignment of `prefs.git_local_only`.

### CODE REVIEW

**Comment by janmulder** [comment]
Tested and LGTM.

One thing I noticed, and I'm just wondering if it is intentional. Open local logbook, open cloud, switch online, hit close, and open the local logbook again. The online toggle is disabled, and unselected. This all looks fine to me. Now do again open cloud. And the cloud is opened in online status. I was a little surprised by this, but not saying that this is not correct. 

Now do the same sequence, but before hit close, go offline first. Again go to a local logbook, and reopen the cloud again. It opens now in offline state. 

In other words. The on/offline state is preserved, even when you visit a local logbook. That is what made me wonder.

**Comment by bstoeger** [comment]
This is not intentional, but - as far as I know - inherited from the previous code. I have a larger patch-set, which among other things changes the behavior to always try to go online if switching from non-cloud to cloud.

From your comment I infer that you would expect the cloud to always start in offline mode?

**Comment by janmulder** [comment]
For me, the current state of this PR is fine, so no need to adapt. I was just a little surprised by the behavior, and indeed, as the (disabled) online-toggle was off when working local, I did not expect it to go to online when re-opening the cloud. 

With respect to "to always try to go online if switching from non-cloud to cloud". I'm not sure I would like this, but that is something time will tell, when you push out your change. The main reason I'm not sure, is that I like the manual control over online/offline cloud mode, and a local storage as well.

**Review by neolit123 — APPROVED** [review]
